app/views/search/search_view.rb

```diff
@@ -1,5 +1,16 @@
 module SearchView
 
+  def user_list
+    search_users.map do |user|
+      {name: user.name, id: user.id}
+    end
+  end
+
+    def search_users
+    User.search(search_params)
```

Comment:
The view layer should never talk to the database, the searching should go in the controller :)